### PR TITLE
Updates README.md

### PR DESCRIPTION
**Pull request by bhanvimenghani**
Makes changes to `README.md` to make it consistent with current code changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,31 @@
...
 
-```
-cd kruize-ui
-```
+Additionally Kruize UI can directly be accessed once remote monitoring demo is setup.
 
-### Switch to mvp_demo branch
+In case you want to run UI locally you would need the Cluster IP and Kruize Port number. Make use of the following commands:
+1. Cluster IP : In case of a minikube cluster ```minikube ip```
+2. Kruize Port Number : ```kubectl -n monitoring  get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort 2>/dev/null```
```
Comment (dinogun) [inline README.md:15]: Please convert this to commands that can be copy pasted
```
export CLUSTER_IP=$(minikube ip)
 export KRUIZE_PORT=$(kubectl -n monitoring  get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort 2>/dev/null)
```

Comment (bhanvimenghani) [reply README.md]: done!

File: README.md [hunk 2]
```diff
@@ -1,68 +1,28 @@
...
-```
+This application is built using [Patternfly and React](https://www.patternfly.org/v4/get-started/developers)
 
-### Go to the kruize folder
+## Before You Start
 
-```
-cd kruize-ui
-```
+Kruize UI needs Kruize server running locally in the remote monitoring mode. You can setup Kruize [from here](https://github.com/kruize/kruize-demos/tree/main/remote_monitoring_demo).
```
Comment (dinogun) [inline README.md:9]: I think we can get rid of `remote monitoring mode` as it get confusing for users.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**